src/data-types/objects/creating-objects.js

```diff
@@ -1,7 +1,16 @@
 // TODO: write code in this section to pass the tests. You will need to add new code
 // as well as modify some of the existing code
-const person = null
-const computer = null
+const person = {
+  name: 'Jane',
+  age: 32
+}
+const computer = {
+  form: 'laptop',
+  specs: (
```

Comment:
there should not be an ( and ) brackets here. To define an object i just need { ... }, so specs should be: